### PR TITLE
Promote the preview to main branch

### PR DESCRIPTION
**Pull request by mtulio**
This PR is a result of the development of the preview release of openshift-tests plugin for OpenShift Provider certification.

Almost all of those files were already reviewed by PRs from repos [[new]](https://github.com/redhat-openshift-ecosystem/provider-certification-plugins/pulls?q=is%3Apr+is%3Aclosed)[[old]](https://github.com/openshift/provider-certification-tool/pulls?q=is%3Apr+is%3Aclosed) created along the development lifecycle of the preview release.

Blocked by https://github.com/redhat-openshift-ecosystem/provider-certification-plugins/pull/18 (last cleaner)

### CODE REVIEW

File: openshift-tests-provider-cert/hack/plugin-sample.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  source-url: https://github.com/redhat-openshift-ecosystem/provider-certification-plugins/
+  skipCleanup: true
+spec:
+  name: plugin
+  image: quay.io/mrbraga/openshift-tests-provider-cert:devel
+  imagePullPolicy: Always
+  priorityClassName: system-node-critical
+  volumeMounts:
+  - mountPath: /tmp/sonobuoy/results
+    name: results
```
Comment (mtulio) [inline openshift-tests-provider-cert/hack/plugin-sample.yaml:47]: TODO check if we can remove it, it should be mounted by default on all containers within pod on environment deployed with sonobuoy v0.55.0+.

**Comment by mtulio** [comment]
I will create a new release `0.1.0-alpha2` and test it before merge this PR.
/hold

**Comment by mtulio** [comment]
`v0.1.0-alpha2` tests have been completed. Promoting it to `stable` (~1h:58m):

- [2022-06-29T13:19:40-03:00] -> [2022-06-29T15:17:57-03:00]

```bash
$ oc describe pods -n openshift-provider-certification |grep Image:
    Image:         quay.io/ocp-cert/sonobuoy:v0.56.5
    Image:         quay.io/ocp-cert/openshift-tests-provider-cert:v0.1.0-alpha2
    Image:          quay.io/ocp-cert/openshift-tests-provider-cert:v0.1.0-alpha2
    Image:         quay.io/ocp-cert/sonobuoy:v0.56.5
    Image:         quay.io/ocp-cert/openshift-tests-provider-cert:v0.1.0-alpha2
    Image:          quay.io/ocp-cert/openshift-tests-provider-cert:v0.1.0-alpha2
    Image:         quay.io/ocp-cert/sonobuoy:v0.56.5

$ ./openshift-provider-cert-cli-mvp run -w --dedicated
INFO[2022-06-29T13:19:40-03:00] Ensuring proper node label for dedicated mode 
INFO[2022-06-29T13:19:40-03:00] Ensuring the tool will run in the privileged environment... 
INFO[2022-06-29T13:19:41-03:00] Created opct-anyuid ClusterRoleBinding       
INFO[2022-06-29T13:19:41-03:00] Created opct-privileged ClusterRoleBinding   
INFO[2022-06-29T13:19:41-03:00] Running OpenShift Provider Certification Tool... 

Wed, 29 Jun 2022 15:14:37 -03> Global Status: running
JOB_NAME                         | STATUS     | RESULTS    | PROGRESS                  | MESSAGE                                           
openshift-conformance-validated  | running    |            | 3239/3239 (30 failures)   | status=running                                    
openshift-kube-conformance       | complete   |            | 345/345 (0 failures)      | waiting for post-processor...                     
Wed, 29 Jun 2022 15:14:47 -03> Global Status: post-processing
JOB_NAME                         | STATUS     | RESULTS    | PROGRESS                  | MESSAGE                                           
openshift-conformance-validated  | complete   |            | 3239/3239 (30 failures)   | waiting for post-processor...                     
openshift-kube-conformance       | complete   |            | 345/345 (0 failures)      | waiting for post-processor...                     
INFO[2022-06-29T15:14:47-03:00] Waiting for post-processor...                
Wed, 29 Jun 2022 15:17:57 -03> Global Status: complete
JOB_NAME                         | STATUS     | RESULTS    | PROGRESS                  | MESSAGE                                           
openshift-conformance-validated  | complete   | failed     | 3239/3239 (30 failures)   | Total tests processed: 1460 (1413 pass / 47 failed)
openshift-kube-conformance       | complete   | failed     | 345/345 (0 failures)      | Total tests processed: 476 (474 pass / 2 failed)  
INFO[2022-06-29T15:17:57-03:00] The execution has completed! Use retrieve command to collect the results and share the archive with your Red Hat partner. 

$ ./openshift-provider-cert-cli-mvp retrieve
INFO[2022-06-29T15:26:04-03:00] Collecting results...                        
INFO[2022-06-29T15:26:13-03:00] Results saved to /path/to/202206291619_sonobuoy_dd3f8c9f-98a3-4711-b988-a8e9b025ab9f.tar.gz 
INFO[2022-06-29T15:26:13-03:00] Use the results command to check the certification test summary or share the results archive with your Red Hat partner. 

```

**Comment by rvanderp3** [comment]
LGTM, i'll defer to @bostrt to give the final lgtm

**Comment by bostrt** [comment]
/lgtm

The 2 unresolved comments seem like they can be followed-up on later. Its exciting to see this moving to main!

**Comment by bostrt** [comment]
@mtulio Just realized the OWNERS file needs an update. Can you fix?

**Review by bostrt — APPROVED** [review]

**Pull request merged**